src/System.Runtime.InteropServices.RuntimeInformation/src/OSPlatform.cs

```diff
@@ -7,19 +7,21 @@ public struct OSPlatform : IEquatable<OSPlatform>
     {
         private readonly string _osPlatform;
 
-        private const string WindowsName = "WINDOWS";
+        private const string FreeBSDName = "FREEBSD";
         private const string LinuxName = "LINUX";
         private const string OSXName = "OSX";
+        private const string WindowsName = "WINDOWS";
 
-        private static readonly OSPlatform s_windows = new OSPlatform(WindowsName);
+        private static readonly OSPlatform s_freebsd = new OSPlatform(FreeBSDName);
         private static readonly OSPlatform s_linux = new OSPlatform(LinuxName);
         private static readonly OSPlatform s_osx = new OSPlatform(OSXName);
+        private static readonly OSPlatform s_windows = new OSPlatform(WindowsName);
 
-        public static OSPlatform Windows
+        public static OSPlatform FreeBSD
```

Comment:
No, that's not new surface area... it's an implementation detail and is the entire point of your change :wink: